Dockerfile.rocm

```diff
@@ -0,0 +1,45 @@
+ARG XFORMERS_COMPILE_JOBS=128
```

Comment:
Not sure if this is the right place for this file (probably in the examples folder? What do we think?)